ui-tests/cypress/support/pageObjects/administration/fileBrowser/FilesListPage.js

```diff
@@ -55,27 +58,68 @@ export default class FilesListPage extends Content {
                .find(this.filesTable);
   }
 
+  getEmptyFolderAlert() {
+    return this.getContents()
+               .find(this.emptyFolderAlert);
+  }
+
   getTableHeaders() {
     return this.getFilesTable()
                .children(htmlElements.thead)
                .children(htmlElements.tr);
   }
 
+  getUpButton() {
+    return this.getTableHeaders().find(this.upButton);
+  }
+
+  clickUpButton() {
+    this.getUpButton().click();
+  }
+
   getTableRows() {
-    return this.getFilesTable()
-               .children(htmlElements.tbody)
-               .children(htmlElements.tr);
+    return cy.get(`${htmlElements.tbody} ${htmlElements.tr}`);
```

Comment:
The only way I found to reliably wait for the file browser to update when creating a new folder (like in the 'Creating a folder' test), was to use a cy.get here. We can probably change this, once the environment is changed to have action context menu buttons for folders have IDs, like the ones for files do.